### DISCUSSION TITLE
p-editor styles not loading in p-dialog

### DISCUSSION THREAD

**Discussion by nestor1610** [primefaces/community#1664] [PrimeNG]
When I call p-editor outside of p-dialog, it works, but inside it doesn't.
The versions of primeng and quill are 17.13.0 and 1.3.7, respectively.

![image](https://github.com/primefaces/community/assets/37000010/aa598ea8-66a0-412e-88af-b2b99a4c8d53)

**Comment by cetincakiroglu** [comment]
Hi,

Could you please share a stackblitz example so we can identify the issue?
